### ISSUE TITLE
Offline songs vanished 

### ISSUE THREAD

**Issue by Ulludapatha** [vfsfitvnm/ViMusic#1430]
### Steps to reproduce the bug

![Screenshot_2024-01-16-20-46-48-110_it vfsfitvnm vimusic](https://github.com/vfsfitvnm/ViMusic/assets/156586863/efbc6121-f340-435e-a919-5e7863acd6de)

### Expected behavior

There must be so many songs that I have listened.

### Actual behavior

Actually there is none.

### Screenshots/Screen recordings

![Screenshot_2024-01-16-20-46-48-110_it vfsfitvnm vimusic](https://github.com/vfsfitvnm/ViMusic/assets/156586863/ac27238e-6f72-4890-80ab-fdffc4c2c986)

### ViMusic version

0.5.4

### Android version

Android 10

**Comment by bidenfelloffhisbike** [comment]
same problem here. android 13, vimusic version 0.5.4

**Comment by Blitzbombeater** [comment]
Yeah its happened to me 5 times it really needs to be fixed but hey you get what you pay for so you can't be too mad about it but it definitely gets annoying having gigabytes of cached songs and images get cleared
